src/components/questions/MultipleChoiceList.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import RadioButtonRN from 'radio-buttons-react-native';
+import { QuestionOption } from '../../models/QuestionOption';
+import FONTS from '../../assets/fonts';
+import COLORS from '../../assets/colors';
+
+interface RadioButtonData {
+  label: string;
+}
+
+const MasterContainer = (props: { questionOption: QuestionOption }) => {
+  const [selectedOption, setSelectedOption] = useState<string>();
+  const multipleChoiceQuestions = props.questionOption.extraData as unknown as string[];
```

Comment:
Is de `as unknown` nodig hier?

Reply:
Jup, waar het goed voor is dunno haha, maar visual studio wilt t, anders krijg je rode lijntjes

Reply:
Ah zal het zijn omdat het type van extraData JSON is? Maar dit wat je nu hebt werkt dus gewoon?

Reply:
Ja ik denk omdat het type JSON heul globaal is, dat die m eerst naar een unknown moet casten voordat ie naar een string array kan. Dit werkt idd gewoon, heb t getest met de questionnaireOptionmock en alles werkte als verwacht (Heb ook screenshots toegevoegd in de PR ;) )

Reply:
Yes good soup